tests/seed-string.test.ts

```diff
@@ -71,7 +71,7 @@ it.each<[string, JSONSchema7, string | RegExp]>([
   [
     'returns a random "date"',
     { type: 'string', format: 'date' },
-    /2024-11-\d{2}/,
+    /2024-\d{2}-\d{2}/,
```

Comment:
These are really annoying. I believe faker has a bug regarding `seed()`.

Reply:
Maybe it's locale related, I didn't dig it